presto-main/src/test/java/com/facebook/presto/operator/GroupByHashYieldAssertion.java

```diff
@@ -109,11 +109,13 @@ public static GroupByHashYieldResult finishOperatorWithYieldingGroupByHash(List<
 
             // add a page and verify different behaviors
             operator.addInput(page);
-            // get output to consume the input
+
+            // consume the input by getOutput() or needsInput()
             Page output = operator.getOutput();
             if (output != null) {
                 result.add(output);
             }
+            boolean fNeedsInput = operator.needsInput();
```

Comment:
Do we need to trigger `needsInput()` here?

Reply:
Also, the variable name `fNeedsInput` looks strange.